src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java

```diff
@@ -263,31 +263,6 @@ public void testDeleteAllCatalogObjectsGrantsAssignedToABucket() {
         verify(catalogObjectGrantRepository, times(1)).delete(dbUserObjectGrants);
     }
 
-    @Test
-    public void testDeleteAllCatalogObjectGrants() {
```

Comment:
why did you remove this test ?

Reply:
I removed the function that it was testing.

Reply:
The function was deleting grants when the object was deleted. Since I changed this mechanism by adding the "cascade" the functions is no longer needed. I removed it then and remove its test.

Reply:
ok